lakefile.lean

```diff
@@ -7,17 +7,17 @@ open Lake DSL
 ## Mathlib dependencies on upstream projects
 -/
 
-require "leanprover-community" / "batteries" @ git "main"
+require "leanprover-community" / "batteries" @ git "nightly-testing"
 require "leanprover-community" / "Qq" @ git "master"
-require "leanprover-community" / "aesop" @ git "master"
-require "leanprover-community" / "proofwidgets" @ git "v0.0.61" -- ProofWidgets should always be pinned to a specific version
+require "leanprover-community" / "aesop" @ git "nightly-testing"
+require "leanprover-community" / "proofwidgets" @ git "v0.0.59" -- ProofWidgets should always be pinned to a specific version
```

Comment:
I don't think this is what we want...

Reply:
Yep, this got merged incorrectly from the master branch created this morning, before I pushed the mathlib update.
```suggestion
require "leanprover-community" / "proofwidgets" @ git "v0.0.61" -- ProofWidgets should always be pinned to a specific version
```